soc/arm/rpi_pico/rp2/soc.c

```diff
@@ -28,19 +28,6 @@ LOG_MODULE_REGISTER(soc, CONFIG_SOC_LOG_LEVEL);
 
 static int rp2040_init(void)
```

Comment:
So now this function does nothing, it can be removed (with the SYS_INIT)

Reply:
Removed it.